### PR TITLE
GitHub Settings update

### PR DESCRIPTION
**Pull request by AleksandarCole**
Updated docs to reflect the new options for Semaphore <> GH connection.

### CODE REVIEW

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 1]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
 - Organizations that you are a member of, and the level of access that Semaphore
   2.0 has to these organizations.
 
-**If the page is not showing Semaphore 2.0 application on GitHub, and you
+If the page is not showing Semaphore 2.0 application on GitHub, and you
 previously logged in to Semaphore 2.0 with the GitHub account that you are
-currently using, then please contact Semaphore support.**
+currently using, please check your [account settings](https://me.semaphoreci.com/account) page.  
+Under "Repository access" sections, click on grayed out checkmarks to reauthorize Semaphore app.
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:19]: > Under "Repository access" sections

why sections? Maybe go for:

> Under the "Repository access" section

Comment (Ivana-Markovic) [reply docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:19]: to reauthorize the Semaphore app.

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 2]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
...
+
+**If [Semaphore 2.0 application page on GitHub](https://github.com/settings/connections/applications/328c742132e5407abd7d) is still not showing Semaphore 2.0 application
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:21]: not showing the Semaphore 2.0 application,

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 3]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
...
 When you sign up with your GitHub account, Semaphore is asking only for your
 email. If you want to grant additional permissions, that is to let Semaphore
 2.0 access public and/or private repositories that you have access to, please
-visit:
+visit [account settings](https://me.semaphoreci.com/account) page.
 
-```
-https://{ORGANIZATION_NAME}.semaphoreci.com/projects/scopes/github
-```
+To grant Semaphore 2.0 access to public or private repositories click on grayed out checkmark to go through authorization process.
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:32]: To grant Semaphore 2.0 access to public or private repositories, click on the grayed out checkmark to go through authorization process.

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 4]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
...
+
+If Semaphore 2.0 already has access, checkmark will be green.
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:34]: ...access, the checkmark will be green.

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 5]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
...
 
 ## Grant access to organization repositories
 
 On the Semaphore 2.0 application page you can also request from a GitHub organization admin
 to grant Semaphore 2.0 access to its repositories. If you are an admin, then you can do
 that yourself.
 
+## Check deploy key health
+When Semaphore 2.0 project is created a deploy key is generated on GitHub.
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:43]: When a Semaphore 2.0 project...

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 6]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
...
 
+## Check deploy key health
+When Semaphore 2.0 project is created a deploy key is generated on GitHub.  
+One deploy key is generated on GitHub repository per each Semaphore 2.0 project connected to that repository.  
+There could be several reasons why deploy key is not available anymore: 
+- It was manually removed from the repository;
+- Original project creator revoked GitHub access to Semaphore app;
+- Original project creator does not have access to that GitHub repository anymore.
+
+If deploy key is broken Semaphore will not be able to check out code from that repository and job log might display following error:
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:50]: If a deploy key is broken, Semaphore will not be able to check out the code from that repository and the job log might display the following error:

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 7]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
...
+``` yaml
+git@github.com: Permission denied (publickey).
+fatal: Could not read from remote repository.
+```
+To check the status of deploy key go to project settings on Semaphore 2.0 and find GitHub settings.
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:55]: To check the status of deploy key, go to the project settings on Semaphore 2.0 and find GitHub settings.

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 8]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
...
+- Original project creator does not have access to that GitHub repository anymore.
+
+If deploy key is broken Semaphore will not be able to check out code from that repository and job log might display following error:
+``` yaml
+git@github.com: Permission denied (publickey).
+fatal: Could not read from remote repository.
+```
+To check the status of deploy key go to project settings on Semaphore 2.0 and find GitHub settings.  
+Green check mark next to "Deploy Key" means that Semaphore found a valid deploy key.  
+If deploy key is broken a red "X" icon will be shown instead.
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:57]: If the deploy key is broken, a red "X" icon will be shown instead.

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 9]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
...
+If deploy key is broken Semaphore will not be able to check out code from that repository and job log might display following error:
+``` yaml
+git@github.com: Permission denied (publickey).
+fatal: Could not read from remote repository.
+```
+To check the status of deploy key go to project settings on Semaphore 2.0 and find GitHub settings.  
+Green check mark next to "Deploy Key" means that Semaphore found a valid deploy key.  
+If deploy key is broken a red "X" icon will be shown instead. 
+
+To regenerate deploy key, click "Repair" next to the red "X" icon. This should delete broken deploy key if it still exists and generate a new one.
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:59]: This should delete the broken deploy key if it still exists and generate a new one.

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 10]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
...
+```
+To check the status of deploy key go to project settings on Semaphore 2.0 and find GitHub settings.  
+Green check mark next to "Deploy Key" means that Semaphore found a valid deploy key.  
+If deploy key is broken a red "X" icon will be shown instead. 
+
+To regenerate deploy key, click "Repair" next to the red "X" icon. This should delete broken deploy key if it still exists and generate a new one.  
+
+## Check webhook health
+Semaphore uses GitHub webhook to detect updates to repository and trigger builds.  
+If webhook is deleted on GitHub, Semaphore will not be able to detect updates and will not run builds.
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:63]: If a webhook is deleted on GitHub,

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 11]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
...
+Green check mark next to "Deploy Key" means that Semaphore found a valid deploy key.  
+If deploy key is broken a red "X" icon will be shown instead. 
+
+To regenerate deploy key, click "Repair" next to the red "X" icon. This should delete broken deploy key if it still exists and generate a new one.  
+
+## Check webhook health
+Semaphore uses GitHub webhook to detect updates to repository and trigger builds.  
+If webhook is deleted on GitHub, Semaphore will not be able to detect updates and will not run builds.  
+
+To check the status of webhook go to project settings on Semaphore 2.0 and find GitHub settings.
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:65]: To check the status of a webhook, go...

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 12]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
...
+
+To regenerate deploy key, click "Repair" next to the red "X" icon. This should delete broken deploy key if it still exists and generate a new one.  
+
+## Check webhook health
+Semaphore uses GitHub webhook to detect updates to repository and trigger builds.  
+If webhook is deleted on GitHub, Semaphore will not be able to detect updates and will not run builds.  
+
+To check the status of webhook go to project settings on Semaphore 2.0 and find GitHub settings.  
+Green check mark next to "Webhook" means that Semaphore managed to connect to GitHub repository.  
+If webhook is broken a red "X" icon will be shown instead.
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:67]: If the webhook is broken, a red "X" icon will be shown instead.

File: docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md [hunk 13]
```diff
@@ -13,28 +13,62 @@ This web page shows you:
...
+
+## Check webhook health
+Semaphore uses GitHub webhook to detect updates to repository and trigger builds.  
+If webhook is deleted on GitHub, Semaphore will not be able to detect updates and will not run builds.  
+
+To check the status of webhook go to project settings on Semaphore 2.0 and find GitHub settings.  
+Green check mark next to "Webhook" means that Semaphore managed to connect to GitHub repository.  
+If webhook is broken a red "X" icon will be shown instead. 
+
+To regenerate webhook, click "Repair" next to the red "X" icon. This should generate a new webhook and repair connection between Semaphore and GitHub.
```
Comment (Ivana-Markovic) [inline docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md:69]: To regenerate the webhook,

**Comment by AleksandarCole** [comment]
@Ivana-Markovic thanks for the review and suggestions. I updated all changes you suggested and a few similar ones, can you do a quick check again?

**Comment by Ivana-Markovic** [comment]
Sorry, it seems I am a bit late to check again, this has been merged 🙂

**Pull request merged**